action.yml

```diff
@@ -0,0 +1,48 @@
+name: poutine - GitHub Actions SAST
+description: |
+  BoostSecurity.io’s poutine detects vulnerabilities and misconfigurations in your GitHub Actions workflows.
+  Designed to streamline security analysis, poutine scans your repository’s CI/CD pipelines.
+  It offers insights to secure your software supply chain efficiently.
+inputs:
+  path:
+    description: 'Path to the local repository to analyze.'
+    required: true
+    default: '.'
+runs:
+  using: composite
+  steps:
+    - name: Download and Setup
+      id: setup
+      shell: bash
+      env:
+        PINNED_VERSION: v0.9.3
+        PINNED_ARCH: poutine_Linux_x86_64.tar.gz
```

Comment:
This is fine for now. Having support for other arch would be nice. I was thinking we should probably use the Docker image for the GitHub Action since it's more portable and should behave better on all types of runners. It's easier to pin a single manifest too, no need for bash logic that determines the architecture, etc.